week1/credit/credit.c

```diff
@@ -0,0 +1,50 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    long cardNumber = get_long("Number: ");
+    long number = cardNumber;
+    int sum = 0;
```

Comment:
This variable name should be changed to `sumOfCardDigits` or `total` think a better name for this. (Optional)